configs/fp16/metafile.yml

```diff
@@ -1,30 +1,30 @@
 Collections:
   - Name: FP16
     Metadata:
-      Training Data: ImageNet
-      Training Resources: 8x V100 GPUs
+      Training Data: ImageNet-1k
```

Comment:
Is it necessary to replace ImageNet with ImageNet-1k

Reply:
Fixed. thx

Reply:
Maybe not, but I just fellow the example in PWC Model-index Format.

Reply:
@ZwwWayne please have a look